data-raw/process_data_sets.R

```diff
@@ -272,3 +272,11 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+
+## Adding international powerlifting data
+ipf_lifts <- readr::read_csv("https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2019/2019-10-08/ipf_lifts.csv") %>% 
```

Comment:
Look at the formatting of the data cleaning in the above examples. The variable should be on one line and so on.